### DISCUSSION TITLE
Production ready images

### DISCUSSION THREAD

**Discussion by volnei** [vercel/turborepo#4281] [Help] [answered]
I'm very confused about how to generate build ready packages from turborepo. There are few examples on the web that is not consistent at all. Can someone share your experiences with build and deploying production images?

**Comment by anthonyshew** [comment, marked as answer]
When you say "images," I'm going to assume that means Docker images. Check out our Docker documentation here: https://turbo.build/repo/docs/handbook/deploying-with-docker
